dataactvalidator/scripts/loadFSRS.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import func
+
+from dataactcore.models.baseInterface import databaseSession
+from dataactcore.models.fsrs import FSRSAward
+from dataactvalidator.fsrs import retrieveAwardsBatch
+
+def loadFSRS(minId):
+    """Fetch and store a batch of Award/Sub-Award records, replacing previous
+    entries."""
+    awards = list(retrieveAwardsBatch(minId))
+    ids = [a.id for a in awards]
+    with databaseSession() as sess:
```

Comment:
Ah yes, thanks for pointing this out. I found the database session code is rather complex, with several "interfaces", shared static fields, and connection closing on object deletion. Rather than than trying to figure that out, I went with a context manager for simplicity. I can swap it out for the existing approach -- I just didn't (still don't, entirely) know what that approach was.